grafana-plugin/src/declare/index.d.ts

```diff
@@ -1 +0,0 @@
-declare module 'slack-markdown';
```

Comment:
`slack-markdown` is not mentioned in `package.json` + we already have a `src/index.d.ts`